internal/config/testdata/example-parameters-string.txt

```diff
@@ -1 +1 @@
-{"csi.storage.k8s.io/pod.name":"nginx-secrets-store-inline","csi.storage.k8s.io/pod.namespace":"test","csi.storage.k8s.io/pod.uid":"9aeb260f-d64a-426c-9872-95b6bab37e00","csi.storage.k8s.io/serviceAccount.name":"default","objects":"array:\n  - |\n    objectPath: \"v1/secret/foo1\"\n    objectName: \"bar1\"\n    objectVersion: \"\"\n  - |\n    objectPath: \"v1/secret/foo2\"\n    objectName: \"bar2\"\n    objectVersion: \"\"\n","roleName":"example-role","vaultAddress":"http://vault:8200","vaultSkipTLSVerify":"true"}
\ No newline at end of file
+{"csi.storage.k8s.io/pod.name":"nginx-secrets-store-inline","csi.storage.k8s.io/pod.namespace":"test","csi.storage.k8s.io/pod.uid":"9aeb260f-d64a-426c-9872-95b6bab37e00","csi.storage.k8s.io/serviceAccount.name":"default","objects":"- secretPath: \"v1/secret/foo1\"\n  objectName: \"bar1\"\n  method: \"GET\"\n- secretPath: \"v1/secret/foo2\"\n  objectName: \"bar2\"","roleName":"example-role","vaultAddress":"http://vault:8200","vaultSkipTLSVerify":"true"}
```

Comment:
Will prettifying this file make a difference? Would be nice if it was a bit more readable.

Reply:
The original intention was to keep it as close as possible to the real input, but prettifying doesn't break anything, and we have the integration tests for the real thing, so good idea 👍